lib/src/iam/signin.rs

```diff
@@ -145,6 +145,8 @@ pub async fn sc(
 									id: Some(rid.to_raw()),
 									..Claims::default()
 								};
+								// Log the authenticated scope info
+								trace!("Signing in to scope `{}`", sc);
```

Comment:
Wondering if we want these. Namespace can work because that isn't strictly confidential? Database, I am a bit dubious about. Scope even more so.
For 1.0 it's probably ok-ish, but for cloud we definitely want to start moving towards not exposing user activity/user security. We can keep it in for now, but we need to be aware that some people would consider this a risk on their infra, and on our cloud.

Reply:
Ok good point. It doesn't expose user details or auth info, only the namespace/database/scope name, which I think might be in the logs already (in DEBUG). Let's check with @sgirones.